locations/exporters/geojson.py

```diff
@@ -33,6 +33,8 @@
     ("image", "image"),
     ("brand", "brand"),
     ("brand_wikidata", "brand:wikidata"),
+    ("operator", "operator"),
+    ("operator_wikidata", "operator:wikidata"),
```

Comment:
Should we consider just doing `operator:wikidata`? `brand:wikidata` is still one of my biggest typos when writing spiders.

Reply:
Oh I guess we can't since it's a variable.